src/client/utils/browser.js

```diff
@@ -22,40 +22,56 @@ function calculateBrowserAndVersion (userAgent) {
     };
 }
 
+
+export var isIE11;
```

Comment:
@miherlosev So? Isn't this is an argument for the utility object which is initialized on `require`?